clients/drcachesim/reader/v2p_reader.cpp

```diff
@@ -103,20 +102,19 @@ v2p_reader_t::create_v2p_info_from_file(std::istream &v2p_file, v2p_info_t &v2p_
 
         found = line.find(physical_address_key);
         if (found != std::string::npos) {
-            error_str = get_value_from_line(line, value);
-            if (!error_str.empty())
-                return error_str;
-            addr_t physical_address = static_cast<addr_t>(value);
-            if (virtual_address == 0) {
-                error_ss << "ERROR: no corresponding " << virtual_address_key << " for "
```

Comment:
So does this fire for a real public trace?  So we need to put out a new 11.3 release?

Reply:
It does, but we needed a new release anyway because of #7136.